### PR TITLE
Added stubs for abrt

### PR DESCRIPTION
**Pull request by tkolhar**
please review it
thanks

### CODE REVIEW

**Comment by elyezer** [comment]
Please follow this format:

```python
"""
"""@Test: a crashed program and abrt reports are send

@Feature: Abrt

@Setup: abrt

@Steps: start a sleep process in background, kill it send the report using
smart-proxy-abrt-send

@Assert: A abrt report with ccpp.* extension created under /var/tmp/abrt

@Status: Manual

"""
```

**Comment by elyezer** [comment]
You can refer to https://gist.github.com/elyezer/b6a68d39adcec098d6a2 for more information.

**Comment by elyezer** [comment]
Still not matching the format, blank lines are relevant and the `@test` should be moved to front of `"""`, like:

```python
"""@test: my test description

...

"""
```

**Comment by Ichimonji10** [comment]
Execute `make docs` to generate documentation locally. Doing so will allow you to see if any new errors are generated and allow you to view the generated documentation locally as a sanity check. It's a decent way to learn reStructuredText.

**Comment by tkolhar** [comment]
@Ichimonji10  tried make docs . no new errors generated

**Comment by Ichimonji10** [comment]
@tkolhar, it's a good start to know that no new errors are generated. But read this again (emphasis added):

> [`make docs`] will allow you to see if any new errors are generated _and allow you to view the generated documentation locally_

The point is that I want you to learn about reStructuredText by looking at the generated documentation. (And, ideally, by reading portions of the spec when you get a chance.) @elyezer and I are just giving you fish when we tell you to fix your docstrings by adding in whitespace, un-indenting code or whatever.

**Comment by tkolhar** [comment]
@Ichimonji10  @elyezer I will definitely go through  reStructuredText tutorial
requesting you to please consider the PR this time
Also, if you could forward some documentation related to the topic
thanks a lot for your guidance

**Comment by Ichimonji10** [comment]
@tkolhar, if you can make the text of this PR match what elyezer has outlined [here](https://github.com/SatelliteQE/robottelo/pull/1830#issuecomment-69905892), that would be great. Importantly:

* Blank lines are important. They separate paragraphs.
* Text should be left-aligned. Do not indent text unless you have a specific reason.

Here is a reST [primer](http://docutils.sourceforge.net/docs/user/rst/quickstart.html), [quick reference](http://docutils.sourceforge.net/docs/user/rst/quickref.html) and [specification](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html).

**Comment by Ichimonji10** [comment]
Thanks for adding the whitespace into the test stubs, @tkolhar. Can you make a few more changes? For example, take this:

    @Steps: 1.) start a sleep process in background, kill it
                send the report using smart-proxy-abrt-send

And rewrite it as so:

    @Steps:

    1. start a sleep process in background, kill it send the report using
    smart-proxy-abrt-send

For an idea why, see a [before](http://rst.ninjs.org/?n=14367a78d822a6b21abda4fba247ccda&theme=basic) and [after](http://rst.ninjs.org/?n=a59a633efb8b3e1a26de6fe65319065d&theme=basic). For more info, see the section of the quick reference regarding [enumerated lists](http://docutils.sourceforge.net/docs/user/rst/quickref.html#enumerated-lists).

**Comment by elyezer** [comment]
I have update the list format on my gist too.

Will check if testimony can parse them, and if not will create a PR for it.

**Comment by Ichimonji10** [comment]
Thanks for checking that out, @elyezer. I appreciate it!

File: tests/foreman/cli/test_abrt.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+        @Setup: abrt
+
+        @Steps:
+
+        1. start a sleep process in background, kill it send the report using
+        smart-proxy-abrt-send
+
+        @Assert: A abrt report with ccpp.*
+                 extension created under /var/tmp/abrt
```
Comment (elyezer) [inline tests/foreman/cli/test_abrt.py]: Missed un-indenting this, example follows:

```python
@Assert: A abrt report with ccpp.* extension created under
/var/tmp/abrt
````

**Comment by elyezer** [comment]
ACK, pending comments.

**Comment by elyezer** [comment]
ACK

**Comment by Ichimonji10** [comment]
ACK

**Pull request merged**